PSReadLine/PSReadLineResources.Designer.cs

```diff
@@ -10,8 +10,8 @@
 
 namespace Microsoft.PowerShell.PSReadLine {
     using System;
-    
-    
+
```

Comment:
Please revert the white space changes, only keep the addition of the new resource strings.

Reply:
reverted